src/utils/helpers/update-profile.js

```diff
@@ -72,6 +72,14 @@ export const schema = yup.object().shape({
          'После символа @ должно быть что-то написано.',
          (value) => valueIsNotEmpty(value.split('@')[1])
       ),
+   password: yup
```

Comment:
Could you add validation for check: includes password one capital letter and one specific symbol or not.